web/modules/custom/joinup_core/templates/statistics-block.html.twig

```diff
@@ -33,6 +23,16 @@
     </a>
     <div class="stats__description">Collaborate around thematics</div>
   </div>
+  <div class="stats__wrapper stats__wrapper__solutions">
```

Comment:
Please include a translate filter to strings in this template to make them accessible for localization.